Add the behaviorbot Welcome bot

nsryan2 opened the pull request:
This PR adds a file congif.yml to the .github folder. This is the bot that @kkiesling found on YT's repository, but the messages have been adapted to better fit PyNE. I'm not completely confident that it works, so I made this a draft.

nsryan2 committed:
Create the welcome bot

@kkiesling found this bot in YT, and it seems like a useful addition to PyNE as we can use it to direct people towards best practices for issues and PRs.

kkiesling commented on .github/config.yml:
```diff
@@ -0,0 +1,28 @@
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hi, and welcome to PyNE!  Thanks for opening your first issue.  We recommend that you 
+  include information such as the version of Pyne you're working with, the platform you
```
```suggestion
  include information such as the version of PyNE you're working with (eg, develop branch or a specific version), the platform you
```

kkiesling commented on .github/config.yml:
```diff
@@ -0,0 +1,28 @@
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: >
+  Hi!  Welcome, and thanks for opening this pull request.  We have some guidelines for
```
Should we link to guidelines? Do we have specific guidelines other than it must pass CI?

nsryan2 replied:
We do not have a specific set of guidelines, I forgot to take that out. I can add a template for PRs, and change the line above to reflect that if we'd like more structure.

kkiesling reviewed (commented):
this makes me so happy!

gonuke commented on .github/config.yml:
```diff
@@ -0,0 +1,28 @@
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Hi, and welcome to PyNE!  Thanks for opening your first issue.  We recommend that you 
```
```suggestion
  Hi, and welcome to PyNE!  :wave: Thanks for opening your first issue.  We recommend that you 
```
It's always good to add emojis :smile:

gonuke reviewed (changes requested):
I've enabled the bot to use this once it's merged.

nsryan2 committed:
Apply suggestions from code review

Co-authored-by: Paul Wilson <paul.wilson@wisc.edu>
Co-authored-by: Kalin Kiesling <krkiesling@gmail.com>

nsryan2 committed:
Add PR template

This commit adds a markdown file that provides structure to PRs to the .github folder

nsryan2 committed:
Update the PR and issue message

nsryan2 commented:
> I've enabled the bot to use this once it's merged.

Thank you! I wasn't sure how to do that part.

bam241 commented:
this need to be rebased.

bam241 commented:
you'll need to update the changelog file

nsryan2 committed:
Update CHANGELOG.rst

kkiesling commented on .github/pull_request_template.md:
```diff
@@ -0,0 +1,24 @@
+## Types of changes
```
does this file appear somewhere with the bot? I don't think we need this. I thought we had contributing guidelines online that we could reference.

nsryan2 replied:
I can get rid of this template for sure, so far as I know we don't have specifics on what should go into a PR anywhere on the site. The [building the website](http://pyne.io/devsguide/website.html#procedure-for-modifying-the-website) page has a procedure for modifying the website, but it doesn't have details for PRs

nsryan2 committed:
Delete pull_request_template.md

gonuke commented:
Thanks @nsryan2  - let's see how it works...
